apps/storybook/src/Footer.stories.tsx

```diff
@@ -107,3 +108,50 @@ export const BottomFixed: Story<FooterProps> = ({ ...rest }: FooterProps) => {
 BottomFixed.parameters = {
   docs: { inlineStories: false },
 };
+
+export const CustomContent: Story<FooterProps> = ({ ...rest }: FooterProps) => {
+  const customItem = (
+    <Footer.Item key='copyright'>
+      <SvgSmileyHappyHollow style={{ width: 12, height: 12, marginRight: 4 }} />
+      <span>Powered by Happiness © {new Date().getFullYear()}</span>
+    </Footer.Item>
+  );
+  return (
+    <Footer {...rest}>
+      {(elements) => {
+        return (
+          <Footer.List>
+            {[
+              customItem,
+              ...elements
+                .filter((el) => el.key !== 'copyright')
+                .map((element, index) => {
```

Comment:
nit: would it be more readable to use `flatMap`? especially since we need to return multiple nodes for each element